tests/ssh-lua-rules/suricata.yaml

```diff
@@ -0,0 +1,4 @@
+%YAML 1.1
```

Comment:
do we need this yaml? can just use default?

Reply:
Can be replaced with just `--set default-rule-path=.`